meteor/server/api/playout/playout.ts

```diff
@@ -985,7 +995,7 @@ export namespace ServerPlayoutAPI {
 		triggerWriteAccessBecauseNoCheckNecessary() // tmp
 
 		return runPlayoutOperationWithLock(
-			context,
+			null, // TODO: should security be done?
```

Comment:
In the end? Yes, I think all methods should ideally be access controlled properly, even the callbacks from devices.
But I suppose that it's a bigger issue, let's cross that stream later.

Reply:
This comment was already on one of these methods, and I don't think it work the same way as the other methods, so thought it best to do this for now.